startup/server/export.js

```diff
@@ -338,10 +338,11 @@ WebApp.connectHandlers.use("/admin/export", (req, res, next) => {
     gameDataFields
   );
 
-  const playerFields = [
-    "_id",
-    "id",
-    "urlParams",
+  const playerFields = ["_id"];
+  if (req.query.remove_pii === "false") {
```

Comment:
This should be the opposite, the default is false: 

```suggestion
  if (req.query.remove_pii !== "true") {
```

Reply:
This now becomes:

```suggestion
  if (req.query.include_pii === "true") {
```